packages/js/src/ui/components/primitives/Tabs/TabsPanel.tsx

```diff
@@ -0,0 +1,31 @@
+import { ParentProps, Show } from 'solid-js';
+import { AppearanceKey } from '../../../context';
+import { cn, useStyle } from '../../../helpers';
+import { useTabsContext } from './Tabs';
+
+type TabsPanelProps = ParentProps & {
+  class?: string;
+  value: string;
+  appearanceKey?: AppearanceKey;
+};
+
+export const TabsPanel = (props: TabsPanelProps) => {
+  const style = useStyle();
+  const { activeTab } = useTabsContext();
+
+  return (
+    <Show when={activeTab() === props.value}>
+      <div
+        class={style(
+          props.appearanceKey ?? 'tabsPanel',
+          cn(activeTab() === props.value ? 'nt-block' : 'nt-hidden', props.class)
```

Comment:
Adding a class prop to this means that another appearance key will be added from this component's usage. However, that extra key will not be placed in the correct spot in the class string.

```
<TabsPanel class={style('test', 'nt-p-10')}
```
would result in `tabsPanel nt-block test nt-p-10`. This means that we need to only use the appearanceKey prop for this but it's easy to use this the wrong way. Maybe it's better to introduce a utility inside `style()` which will sort the classnames based on which classes exist on `appearanceKeys` and get rid of the `appearanceKey` props altogether. Let's leave this for another PR, just sharing my thoughts.